pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -441,6 +439,7 @@ stdenv.mkDerivation (finalAttrs: {
      *  External libraries
      */
     (enableFeature withAlsa "alsa")
+    (enableFeature withAribcaption "libaribcaption")
```

Comment:
actually sorry, this shouldn't be enableFeature because ffmpeg 6.0 lacks the flag `--disable-libaribcaption`. Indeed, optionalString should be used here until jellyfin-ffmpeg is updated.

Reply:
But withAribcaption checks for version at least 6.1, I think it is not a problem.

Reply:
again, the problem is that if withAribcaption is false, the flag `--disable--libaribcaption` will be passed, however it's an illegal flag for ffmpeg 6.0

Reply:
Oh, now I understood. Done.